students/10th/ParkMinKyu/js/main.js

```diff
@@ -0,0 +1,35 @@
+const replyBox = document.getElementsByClassName("reply-box")[0];
+const button = document.getElementsByClassName("button-key")[0];
+const inputBar = document.getElementsByClassName("reply-input-bar")[0];
+
+button.addEventListener("click", function () {
+  let reply = document.createElement("li");
+  let replyText = document.createElement("span");
+  let replyValue = inputBar.value;
+  if (replyValue !== "") {
+    replyText.innerHTML = "<b>wecode_10</b>" + " " + replyValue;
+    reply.appendChild(replyText);
+    replyBox.appendChild(reply);
+    inputBar.value = "";
+    return;
+  }
+});
+
+const inputEnter = document.getElementsByClassName("reply-input-bar")[0];
+inputEnter.addEventListener("keyup", function (
+  event
+) /*함수안에 envent 쓴이유*/ {
```

Comment:
가독성을 위해 아래처럼 수정해주세요!
```suggestion
inputEnter.addEventListener("keyup", function (event) {
```

Reply:
수정 완료했습니다.